app/src/composables/use-revisions.ts

```diff
@@ -132,22 +132,7 @@ export function useRevisions(
 						},
 						sort: '-id',
 						limit: 1,
-						fields: [
-							'id',
-							'data',
-							'delta',
-							'collection',
-							'item',
-							'activity.action',
-							'activity.timestamp',
-							'activity.user.id',
-							'activity.user.email',
-							'activity.user.first_name',
-							'activity.user.last_name',
-							'activity.ip',
-							'activity.user_agent',
-							'activity.origin',
-						],
+						fields: ['id'],
```

Comment:
Just make sure I understand this correctly. The extra fields were removed because `created` is only used to assert existence of a revision in `revisions-drawer-detail`, and thus only `id` is needed?

Reply:
Yes! That is correct.